spec/models/guide_search_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe GuideSearch do
+  subject { GuideSearch.search }
+
+  it 'defaults query to "*"' do
+    expect(subject.query).to eq('*')
+  end
+
+  it 'searches for guides associated with crops' do
+    crop = OpenStruct.new(id: 1)
+
+    subject.for_crops(crop)
+
+    expect(subject.filter[:crop_id]).to include(crop.id)
+  end
+
+  it 'searches for guides associated with crops by id' do
+    crop = OpenStruct.new(id: 1)
+
+    subject.for_crops(crop.id)
+
+    expect(subject.filter[:crop_id]).to include(crop.id)
+  end
+
+  it 'searches without a user' do
+    expect(subject.order).to eq(_score: :desc)
+  end
+
+  it 'searches with a user' do
+    user = OpenStruct.new(id: "1")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.